R/pmcmc_parameters.R

```diff
@@ -306,3 +498,44 @@ check_parameters <- function(parameters, type) {
   names(parameters) <- nms
   parameters
 }
+
+# p <- pmcmc_parameters$new(
```

Comment:
can you please remove commented code? If you want to turn this into tests or examples that's fine